Message flow interchange (#3)
inform-health-informatics/emap_documentation: technical_overview/Github_repos.md

# Overview INFORM Github repositories
 
While the "Inform Health Informatics" Github organisation holds a much larger number of Github repositories, currently 
relevant to any developer wanting to contribute to the project are the following. Note that special permissions may be 
required to access those.

| Github repo | Short description | maintainer |
| ----- | ---- | ----|

> | [hoover](https://github.com/inform-health-informatics/hoover) | Source code to "hoover" data from the historic database in the hospital | Jeremy |

Reviewer: ```suggestion
| [hoover](https://github.com/inform-health-informatics/hoover) | Source code to "hoover" data from hospital databases. Publishes to rabbitMQ queue | Jeremy |
```

| [db_validation](https://github.com/inform-health-informatics/db_validation) | Contains code to check integrity and consistency of data across the different databases of the EMAP pipeline. | Aasiyah |

> | [Emap-Interchange](https://github.com/inform-health-informatics/Emap-Interchange) | Defines the EMAP Interchange format, which is utilised for a consistent data representation across the different sources. | Jeremy, Stef |

Reviewer: ```suggestion
| [Emap-Interchange](https://github.com/inform-health-informatics/Emap-Interchange) | Defines the EMAP Interchange format, which is utilised for a consistent data representation across the different sources. | All EMAP RSDG members |
```

> | [Emap-hl7-processor](https://github.com/inform-health-informatics/emap-hl7-processor) | Code to read data from the different HL7 message streams. Publishes message on respective RabbitMQ queue. | Stef |

Reviewer: ```suggestion
| [Emap-hl7-processor](https://github.com/inform-health-informatics/emap-hl7-processor) | Code to read data from the different HL7 message streams. Publishes messages to RabbitMQ queue. | Stef |
```
